src/language-css/parser-postcss.js

```diff
@@ -382,7 +382,7 @@ function parseNestedCSS(node, options) {
 
     if (node.type === "css-atrule") {
       // mixin
-      if (node.mixin) {
+      if (options.parser !== "scss" && node.mixin) {
```

Comment:
The `css` parser should not support `minix` and other stuff because mixins doesn't exists in `css`, yes we use `postcss-less` for parsing pure CSS, but it is not good and in future we should switch from `postcss-less` on built-in css parser inside `postcss`